### PR TITLE
update READEME file

### PR DESCRIPTION
**Pull request by AbdallahMahdyyy**
- adding the description 

- make it descriptive

### CODE REVIEW

**Comment by abdallah-mahdy** [comment]
sounds good to me

File: README.txt [hunk 1]
```diff
@@ -1 +1 @@
-This branch is about learning
+a Dogs website project
```
Comment (abdallah-mahdy) [inline README.txt:1]: good description but we can 
```suggestion
a Dogs website project should be the worest website in the world
```

File: README.txt [hunk 2]
```diff
@@ -1 +1 @@
-This branch is about learning
```
Comment (abdallah-mahdy) [inline README.txt:1]: why did you delete it

Comment (AbdallahMahdyyy) [reply README.txt:1]: because it is not descriptive

File: README.txt (hunk 1 above)
Comment (AbdallahMahdyyy) [reply README.txt:1]: Thanks
